types/svg4everybody/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for svg4everybody 2.1
+// Project: https://github.com/jonathantneal/svg4everybody#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Svg4everybodyOpts } from "svg4everybody";
+export as namespace svg4everybody;
+
+export = svg4everybody;
+
+declare function svg4everybody(options?: Svg4everybodyOpts): void;
```

Comment:
Then replace `Svg4everybodyOpts` on this line with `svg4everybody.Svg4everybodyOpts`.